src/frontend/services/addHistoryService.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+export const addHistoryService = async (token, video) => {
+  try {
+    const { data } = await axios.post(
+      "/api/user/history",
+      { video },
+      {
+        headers: {
```

Comment:
Instead of passing headers for all services, again and again, create an axios instance
Ref: https://axios-http.com/docs/instance